pkg/cloud/cloud_test.go

```diff
@@ -53,6 +53,7 @@ func (tp *testPlatform) getOperatorConfig() config.OperatorConfig {
 			CloudControllerManagerOpenStack: "registry.ci.openshift.org/openshift:openstack-cloud-controller-manager",
 			CloudControllerManagerVSphere:   "registry.ci.openshift.org/openshift:vsphere-cloud-controller-manager",
 			CloudControllerManagerPowerVS:   "quay.io/openshift/origin-powervs-cloud-controller-manager",
+			CloudControllerManagerNutanix:   "registry.ci.openshift.org/openshift:nutanix-cloud-controller-manager",
```

Comment:
I think this should be `quay.io/openshift/origin-nutanix-cloud-controller-manager` as is the format with Azure, not sure why these are mixed 🤔

Reply:
Changed to quay.io/openshift/origin-nutanix-cloud-controller-manager